src/main/java/com/esri/samples/grouplayers/group_layers/GroupLayersSample.java

```diff
@@ -0,0 +1,170 @@
+
+public class GroupLayersSample extends Application {
+
+  private SceneView sceneView;
+
+  @Override
+  public void start(Stage stage) {
+
+    try {
+
+      // set the title and size of the stage and show it
+      StackPane stackPane = new StackPane();
+      Scene fxScene = new Scene(stackPane);
+      stage.setTitle("Group Layers Sample");
+      stage.setWidth(800);
+      stage.setHeight(700);
+
+      // create a JavaFX scene with a stackpane and set it to the stage
+      stage.setScene(fxScene);
+      stage.show();
+
+      // create a scene view and add it to the stack pane
+      sceneView = new SceneView();
+      stackPane.getChildren().add(sceneView);
+
+      // create a scene with a basemap and add it to the scene view
+      ArcGISScene scene = new ArcGISScene();
+      scene.setBasemap(Basemap.createImagery());
+      sceneView.setArcGISScene(scene);
+
+      // set the base surface with world elevation
+      Surface surface = new Surface();
+      surface.getElevationSources().add(new ArcGISTiledElevationSource("http://elevation3d.arcgis.com/arcgis/rest/services/WorldElevation3D/Terrain3D/ImageServer"));
+      scene.setBaseSurface(surface);
+
+      // create different types of layers
+      ArcGISSceneLayer devOne = new ArcGISSceneLayer("https://scenesampleserverdev.arcgis.com/arcgis/rest/services/Hosted/DevA_Trees/SceneServer/layers/0");
+      ArcGISSceneLayer devTwo = new ArcGISSceneLayer("https://scenesampleserverdev.arcgis.com/arcgis/rest/services/Hosted/DevA_Pathways/SceneServer/layers/0");
+      ArcGISSceneLayer devThree = new ArcGISSceneLayer("https://scenesampleserverdev.arcgis.com/arcgis/rest/services/Hosted/DevA_BuildingShell_Textured/SceneServer/layers/0");
+      ArcGISSceneLayer nonDevOne = new ArcGISSceneLayer("https://scenesampleserverdev.arcgis.com/arcgis/rest/services/Hosted/PlannedDemo_BuildingShell/SceneServer/layers/0");
+      FeatureLayer nonDevTwo = new FeatureLayer(new ServiceFeatureTable("https://services.arcgis.com/P3ePLMYs2RVChkJx/arcgis/rest/services/DevelopmentProjectArea/FeatureServer/0"));
+
+      // create a group layer from scratch by adding the layers as children
+      GroupLayer groupLayer = new GroupLayer();
+      groupLayer.setName("Group: Dev A");
+      groupLayer.getLayers().addAll(Arrays.asList(devOne, devTwo, devThree));
+
+      // add the group layer and other layers to the scene as operational layers
+      scene.getOperationalLayers().addAll(Arrays.asList(groupLayer, nonDevOne, nonDevTwo));
+
+      // zoom to the extent of the group layer when the child layers are loaded
+      groupLayer.getLayers().forEach(childLayer ->
+        childLayer.addDoneLoadingListener(() -> {
+          if (childLayer.getLoadStatus() == LoadStatus.LOADED) {
+            sceneView.setViewpointCamera(new Camera(groupLayer.getFullExtent().getCenter(), 700, 0, 60, 0));
+          }
+        })
+      );
+
+      // create a tree view to show the layers in the scene
```

Comment:
specify it's a JavaFX tree view (for new developers), since so far the code has been ArcGIS specific.